e2e/windows/e2e-scenario.sh

```diff
@@ -41,6 +41,19 @@ collect-logs() {
             err "Failed in deleting cse logs in remote storage. Error code is $retval."
         fi
     fi
+
+    array=(azcopy_*)
```

Comment:
Suggest you wrap this code to a function

Reply:
Good suggestion. I think that we can do it later since it needs more time to do testing and there are many places in using the similar logic.
I think that we can use some functions, `azcopy-get`, `azcopy-delete`, etc.

Reply:
@ShiqianTao Created the ADO task.